spec/javascript/sum.test.js

```diff
@@ -0,0 +1,5 @@
+const sum = require('./sum');
+
+test('adds 1 + 2 to equal 3', () => {
+    expect(sum(1, 2)).toBe(3);
+});
```

Comment:
Ah, a beauty of a test there :)

I would just make sure you remember to add proper tests for your react code once you've written some components that are more relevant to the blog.

Reply:
Absolutely, quality will remain in focus!